scripts/participateInAuction.js

```diff
@@ -28,16 +28,16 @@ module.exports = async (callback) => {
       const pool1 = await DxMgnPool.at(await coordinator.dxMgnPool1())
       const pool2 = await DxMgnPool.at(await coordinator.dxMgnPool2())
 
-      const pool1State = (await pool1.currentState()).toNumber()
+      const pool1State = (await pool1.checkForStateUpdate()).toNumber()
```

Comment:
Can this be a `.call` instead of sending a transaction? Otherwise, wed'd have to pay gas for it every single time.

Reply:
Yes of course! Can't believe I forgot that.